core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
I still don't get it. There is no trailing whitespace (I deleted it), and there is an extra blank line. What are you looking for exactly?